src/Roslyn.Diagnostics.Analyzers/Core/DeclarePublicAPIAnalyzer.Impl.cs

```diff
@@ -359,6 +390,49 @@ internal void OnCompilationEnd(CompilationAnalysisContext context)
                 }
             }
 
+            private void ProcessTypeForwardedAttributes(Compilation compilation, Action<Diagnostic> reportDiagnostic, CancellationToken cancellationToken)
+            {
+                var typeForwardedToAttribute = compilation.GetTypeByMetadataName("System.Runtime.CompilerServices.TypeForwardedToAttribute");
```

Comment:
As a policy, I don't when writing analyzers. Although in this case you _could_ because this is a pretty core type, it then means you're blurring "the framework my analyzer builds against" versus "the framework which contains the types my analyzer is trying to analyze". If we were checking some other type _not_ in the framework you definitely wouldn't, and I think it's wise to illustrate the general practice because people might be using these as code samples for themselves.